.github/workflows/updates.yml

```diff
@@ -40,7 +45,8 @@ jobs:
           key: ${{ runner.os }}-${{ runner.arch }}-${{ env.SUI_TAG }}
       - name: Install sui
         if: steps.cache-sui-restore.outputs.cache-hit != 'true'
-        run: cargo install --locked --git https://github.com/MystenLabs/sui.git --tag $SUI_TAG --debug sui
+        run: |
+          cargo install --locked --git https://github.com/MystenLabs/sui.git --tag $SUI_TAG --debug sui
```

Comment:
This could take a very long time. Would it be possible to just download the binary from GitHub releases here?

Reply:
Unfortunately, the release build is not suitable for computing test coverage (which we do in `scripts/move_tests.sh`).